packages/client/modules/teamDashboard/components/ProviderRow/StyledButton.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled'
+import FlatButton from '../../../../components/FlatButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+
+const StyledButton = styled(FlatButton)({
```

Comment:
+1 Could get a better name, like in what way is it styled differently than `FlatButton`?